share-extension/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,26 @@
+/* 
+  Localizable.strings
+  eXo
+
+  Created by Nguyen Manh Toan on 9/9/15.
+  Copyright (c) 2015 eXo Platform. All rights reserved.
+*/
+
+
+"All uploads failed" = "All uploads failed";
```

Comment:
Replace keys by ones that follow the pattern specified
https://community.exoplatform.com/portal/g/:spaces:exo_mobile/exo_mobile/wiki/Tech_Spec_-_Hybrid_app#HLocalization